### PR TITLE
Update changelog after prod release

### PR DESCRIPTION
**Pull request by eadpearce**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ## Pre-release
 
+### Implemented enchancements
+
```
Comment (jessgilbert) [inline CHANGELOG.md]: Not every ticket was released was it? Do you need to move others in to pre-release now?

Comment (eadpearce) [reply CHANGELOG.md]: Yeah I was expecting git to complain about merge conflicts... I guess I'll merge and fix them locally

Comment (eadpearce) [reply CHANGELOG.md]: Think this is correct now

Comment (jessgilbert) [reply CHANGELOG.md]: Should these also be in the implemented enhancements? 
```
- XOT-989 - remove Content from breadcrumbs for SOO market sign up
- XOT-968 - Fix security issues raised by latest pen test
```

Comment (jessgilbert) [reply CHANGELOG.md]: ^^ From your message in cross dev about what is being released

Comment (eadpearce) [reply CHANGELOG.md]: Yeah I was gonna fix those with merging master back into dev but.. i think it'd be easier if i just do it here

**Pull request merged**